custom-recipes/text-extraction/recipe.json

```diff
@@ -38,6 +38,12 @@
             "label" : "Extract text chunks",
             "type": "BOOLEAN",
             "description": "Output one row by unit of document. A unit can be a page in a PDF file, a section in a Markdown file..."
+        },
+        {
+            "name": "natural_format",
+            "label" : "Metadata in natural format",
+            "type": "BOOLEAN",
+            "description": "Output the metadata in a natural format instead of JSON."
```

Comment:
We should hide this parameter if not working with chunks, using `visibilityCondition` https://doc.dataiku.com/dss/latest/plugins/reference/params.html#refdoc-plugin-parameter-conditional-parameters